app/models/room.rb

```diff
@@ -18,12 +18,20 @@ def init
     self.outer_shape ||= Polyline.new # if no outer shape exists yet, create an empty one
   end
 
+  def to_string
+    "#{house}-#{floor}.#{number} #{full_name}"
+  end
+
   def to_geojson
     walls.map(&:to_geojson) +
       point_of_interests.map(&:to_geojson) +
       [ outer_shape.to_geojson.merge({ properties: { class: "outer-shape" } }) ]
   end
 
+  def self.searchable_attributes
+    %w[house floor number full_name]
+  end
+
   def display_name
     full_name || number
   end
```

Comment:
What is the difference between to_string and display_name? What does `full_name` contain?

Reply:
Well in this case they even have the same meaning, so maybe you could merge them here?